engine/src/main/java/org/terasology/logic/console/commands/WorldCommands.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.logic.console.commands;
+
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.entity.internal.EngineEntityPool;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.logic.chat.ChatMessageEvent;
+import org.terasology.logic.common.DisplayNameComponent;
+import org.terasology.logic.console.commandSystem.annotations.Command;
+import org.terasology.logic.console.commandSystem.annotations.CommandParam;
+import org.terasology.logic.console.commandSystem.annotations.Sender;
+import org.terasology.network.ColorComponent;
+import org.terasology.registry.In;
+import org.terasology.rendering.nui.Color;
+import org.terasology.world.internal.WorldInfo;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RegisterSystem
+public class WorldCommands extends BaseComponentSystem {
+
+    @In
+    private EntityManager entityManager;
+
+
+    @Command(shortDescription = "Get information about different worlds and " +
+            "entities present in each pool", runOnServer = true)
+    public String getUniverseInfo() {
+        int worldCount = entityManager.getWorldPools().size();
+        StringBuilder message = new StringBuilder(100);
```

Comment:
what's 100?
Pretty sure you can make a string builder without defining capacity explicitly